### PR TITLE
(Enhancement + Code) Manager doesn't show users fullname in header #1…

### LINKED ISSUE #12457: (Enhancement + Code) Manager doesn't show users fullname in header
Hey guys,
I was wondering why the manager doesn't actually shows the fullname as "Username" of the user who is logged into the manager (next to the gravatar-image), so I solved this issue by myself.

If the fullname-field ist empty, there is a fallback to the actual "username":
2015-05-26_00h31_23

But if the fullname-field is filled, it gets used, to show the much cleaner fullname:
2015-05-26_00h31_03

Replace the function "setPlaceholders" (starting at line 78) in the file "manager/controllers/default/header.php" with the following to get it work:
==============================================================

    public function setPlaceholders()
    {
        
        $userProfile = $this->modx->user->getOne('Profile');
        $username = $userProfile->get('fullname');
        if(empty($username)) $username = $this->modx->getLoginUserName();
        $placeholders = array(
            'username' => $username,
            'userImage' => $this->getUserImage(),
        );

        $this->controller->setPlaceholders($placeholders);
    }

==============================================================

Many greeting from germany
Jens

### PR DESCRIPTION
**Pull request by JaWe09**
### What does it do ?

In #12457 @JensWolff provides a method for showing fullname instead of login name. I've added this method with system setting support.
### Why is it needed ?

Because some other people and i think  that it is a good idea :) and because modx posibilities and flexibility is awesome and should be developed further :) 
### Related issue(s)/PR(s)

fixes #12457
#12523

@Mark-H i hope thats better :) if I miss something please let me know.

### CODE REVIEW

File: manager/controllers/default/header.php [hunk 1]
```diff
@@ -77,8 +77,17 @@ public function render()
      */
     public function setPlaceholders()
     {
+
+        if($this->modx->getOption('manager_use_fullname') == true) {
+            $userProfile = $this->modx->user->getOne('Profile');
+            $username = $userProfile->get('fullname');
+        } else {
+            $username = '';
```
Comment (rtripault) [inline manager/controllers/default/header.php]: Am i right to assume that if `manager_use_fullname` is set to `false` neither the username nor the full name would be available ?

Comment (rtripault) [inline manager/controllers/default/header.php]: Nevermind... line 88 takes care of that... time for another coffee (sorry for the noise)

Comment (wuschigesetwas) [inline manager/controllers/default/header.php]: @rtripault no probleme. it's better to make a call than missing a problem :) :+1:

**Comment by Mark-H** [comment]
Thank you for making MODX better @wuschigesetwas  :)

**Comment by JensWolff** [comment]
:+1: :+1: :+1: 
Thanks, guys! :)

**Comment by wuschigesetwas** [comment]
@JensWolff you're welcome :+1: 
@Mark-H  you too. I'm planing to do more on this. I already take a look on issues and will work on it in the future and I'm looking forward to a good cooperation :) :+1:

**Pull request merged**